.github/workflows/ant.yaml

```diff
@@ -174,3 +174,11 @@ jobs:
           fields: repo,message,commit,author,action,eventName,ref,workflow,job
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+  propose:
+    needs: [publish]
+    if: needs.publish.result == 'success'
+    uses: ./.github/workflows/vaot-proposal.yaml
+    secrets: inherit
+    with:
+      published_module_id: ${{ needs.publish.outputs.published_module_id }}
+      published_lua_code_id: ${{ needs.publish.outputs.published_lua_code_id }}
```

Comment:
add dry-run flag here and set it as a repository variable. that'll allow us to quickly turn off if we need to

Reply:
https://github.com/ar-io/ar-io-ant-process/pull/86/commits/31506ad5c6998ea3bf5e51c18fc799e0206c664c